go-controller/pkg/util/util.go

```diff
@@ -85,7 +85,8 @@ func UpdateNodeSwitchExcludeIPs(nodeName string, subnet *net.IPNet) error {
 		if strings.Contains(line, "(k8s-"+nodeName+")") {
 			haveManagementPort = true
 		} else if strings.Contains(line, "("+houtil.GetHybridOverlayPortName(nodeName)+")") {
-			haveHybridOverlayPort = true
+			// we always need to set to false because we do not reserve the IP on the LSP for HO
+			haveHybridOverlayPort = false
```

Comment:
yeah, we should probably fix this afterwards

Reply:
So because we don't set the IP anymore on the port, we always need this to be false. So it is true that it could have just been set to false outside of this line. I was just in a rush so just did it here. The mgmt port from my conversation with @dcbw still needs the logic, because without it we will get a warning from OVN. The logic is:
1) if the port has the IP assigned then we dont need to update the exclude ips (IP is already reserved)
2) if the port has the ip and exclude ips, we will get a warning

Reply:
Yes, mgmt port still needs the logic otherwise we'll get continuous log spew from ovn-northd. Yes, its counter intuitive.

We need to set exclude_ips=XXXX *until there is an LSP with that IP*. But after an LSP with that IP exists we must remove that IP from exclude_ips to prevent the logspam.